cmd/plural/validation.go

```diff
@@ -141,3 +142,10 @@ func repoRoot() error {
 
 	return nil
 }
+
+func latestVersion(fn func(*cli.Context) error) func(*cli.Context) error {
+	return func(c *cli.Context) error {
+		utils.CheckLatestVersion(version)
```

Comment:
We should run this maybe 1 in 10 or 1 in 5 executions, and also ignore entirely when run in the console (there are other cases where we do this)

Reply:
@michaeljguarino 
For random execution: 
 1) should I add some random trigger inside the function?
```
	rand.Seed(time.Now().UnixNano())
	if rand.Intn(5) == 4 {
		utils.CheckLatestVersion(version)
	}

```
 2) save the number of execution in ~/.plural/config.yml to make the decision on when to enable the check


For the console:
1) add CLI global flag with the env variable (https://github.com/pluralsh/plural-cli/blob/master/cmd/plural/plural.go#L418) to disable the `utils.CheckLatestVersion(version)` method
for example:
```
$ PLURAL_DISABLE_LATEST_CHECK=true plural init
```
by default will be false for regular usage.

2) something else maybe?

Reply:
Yeah a random coinflip like that looks perfect.  We should probably add a coinflip function to polly too since it's generally very useful.

I think we should just have a general env var that tells the cli it's in the console, could literally be `PLURAL_CONSOLE=1` or something

Reply:
@michaeljguarino thanks, I will follow your suggestions